e2e/posts-preview.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { test, expect } from "@playwright/test";
+import path from 'path';
+
+import { deletePost, getPostIdInURL, createPostWithFeatureImage } from "./helpers/post";
+
+test.describe('feature image', () => {
+  let postIdsToDelete: string[] = [];
```

Comment:
```suggestion
  const postIdsToDelete: string[] = [];
```
This is never reassigned.